apps/web/__e2e__/sorting-notebooks.test.ts

```diff
@@ -0,0 +1,205 @@
+    orderBy: "descendingOrder",
+    sortBy: "dateCreated"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "week",
+    orderBy: "ascendingOrder",
+    sortBy: "dateCreated"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "week",
+    orderBy: "descendingOrder",
+    sortBy: "dateCreated"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "year",
+    orderBy: "ascendingOrder",
+    sortBy: "dateCreated"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "year",
+    orderBy: "descendingOrder",
+    sortBy: "dateCreated"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+
+  //dateEdited
+  await notebooks.sort({
+    groupBy: "abc",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "abc",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "default",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "default",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "month",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "month",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "none",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "none",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "week",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "week",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "year",
+    orderBy: "ascendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+  await notebooks.sort({
+    groupBy: "year",
+    orderBy: "descendingOrder",
+    sortBy: "dateEdited"
+  });
+  expect(await isLengthCorrect(page)).toBeTruthy();
+});
+
+async function isLengthCorrect(page: Page) {
+  let itemCount = await page.locator(getTestId("list-item")).count();
+  if (itemCount !== 5) return false;
+  return true;
+}
```

Comment:
This is not the right place for this. If you just want the item count, move the method to `BaseViewModel` & name it appropriately.